lightning/src/ln/channel.rs

```diff
@@ -684,6 +684,7 @@ pub(super) struct ChannelContext<Signer: ChannelSigner> {
 	next_holder_htlc_id: u64,
 	next_counterparty_htlc_id: u64,
 	feerate_per_kw: u32,
+	high_priority_feerate_per_kw: Option<u32>,
```

Comment:
Before I go add a lifetime parameter + generic to a bunch of places, just wanna check instead of caching we'd just have a reference to the fee estimator in each channel right?

Reply:
Oh ok, thanks

Reply:
Done